suites/squid/cephfs/tier-3_cephfs_bugs.yaml

```diff
@@ -268,4 +268,11 @@ tests:
       polarion-id: CEPH-83592513
       desc: Verify Client evict -h,--help works without running the function
       abort-on-fail: false
+  - test:
```

Comment:
nit: traceability via git log would be possible if we have them in separate commits. We would also need to add better commit messages.

Reply:
done, removed suite files, will include it another PR